src/backend/marsha/websocket/consumers/video.py

```diff
@@ -175,8 +175,12 @@ async def disconnect(self, code):
         # Leave room group
```

Comment:
In the commit message:

`s/When we disconnect fron a websocket/When we disconnect from a websocket`